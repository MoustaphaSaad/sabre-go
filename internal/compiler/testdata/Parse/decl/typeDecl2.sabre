type (
	x int
	y int
)