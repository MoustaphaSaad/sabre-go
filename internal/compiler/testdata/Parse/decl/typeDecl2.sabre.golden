(GenericDecl type
  (TypeSpec
    (IdentifierExpr IDENTIFIER(x))
    (NamedType IDENTIFIER(int))
  )
  (TypeSpec
    (IdentifierExpr IDENTIFIER(y))
    (NamedType IDENTIFIER(int))
  )
)